Add a Chart for Locust Distributed Load Testing

so0k opened the pull request:
This chart will install Locust master and worker containers.

Please advise on what is required to improve the chart.

Source for container:

- https://github.com/honestbee/distributed-load-testing

Commit:
Add first locust chart draft

so0k commented:
@ k8s-ci-robot I signed it!

so0k commented:
Seems I introduced a problem with the communication from worker to masters when converting to a chart.. if someone could help troubleshoot :(

so0k commented:
I confirmed the docker images (alpine based) work locally with compose... I'm stumped to find out the cause for my issue :(

https://github.com/honestbee/distributed-load-testing#testing-locust-locally

so0k commented:
Found & fixed the error, this chart is ready for review

seanknox commented on incubator/locust/templates/master-deploy.yaml:
```diff
@@ -0,0 +1,67 @@
+        #   periodSeconds: 30
+        #   httpGet:
+        #     path: /
+        #     port: {{ .Values.service.internalPort }}
```
Can you uncomment or remove the livenessProbe section?

seanknox commented on incubator/locust/values.yaml:
```diff
@@ -0,0 +1,36 @@
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
```
Uncomment these too.

seanknox commented:
Hi @so0k, thanks for submitting this. Few changes needed:

1. A chart should be in `stable/` unless it uses alpha APIs (I didn't see any here)
2. Would appreciate a more descriptive `NOTES.txt` for the user. This [drupal example](https://github.com/kubernetes/charts/blob/master/stable/drupal/templates/NOTES.txt) is a good one.
3. Couple minor comments regarding commented out code.

Other than that, this looks LGTM.

Commit:
Move to stable

Commit:
Improve post installation NOTES

Commit:
Enable Health Check on locust master

Commit:
Enable hard limit on Kubernetes Resources

Commit:
Ensure master is ready before starting workers

so0k commented:
@seanknox, thanks for the review!

I made following changes according to your comments:

- [x] Move to stable
- [x] Improve NOTES
- [x] uncommented code

While testing my changes on minikube, I also noticed a small bug with the workers starting before the master, so I've added a work around to the `entrypoint` of the locust image and added another env var to the deployment templates.

so0k commented:
Oops, I accidentally pushed cloudflare datadog chart as well

EDIT: Fixed (rebased and squashed first few commits)

Commit:
Fix quotes for Worker deployment

so0k commented:
Thanks for the fast turnaround
